src/views/ModelPlan/CollaborationArea/Cards/ModelPlanCard/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader
+} from '@trussworks/react-uswds';
+import {
+  GetModelPlanQuery,
+  TaskStatus,
+  useGetModelPlanQuery
+} from 'gql/gen/graphql';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Modal from 'components/Modal';
+import { Avatar } from 'components/shared/Avatar';
+import ShareExportModal from 'components/ShareExport';
+import { formatDateLocal } from 'utils/date';
+import {
+  getLatestModifiedDate,
+  ITSolutionsType,
+  StatusMessageType
+} from 'views/ModelPlan/TaskList';
+import { TaskListStatusTag } from 'views/ModelPlan/TaskList/_components/TaskListItem';
+
+import './index.scss';
+
+type GetModelPlanTypes = GetModelPlanQuery['modelPlan'];
+type OperationalNeedsType = GetModelPlanQuery['modelPlan']['operationalNeeds'][0];
+
+type ModelPlanCardType = {
+  modelID: string;
+  setStatusMessage: (message: StatusMessageType) => void;
+};
+
+const ModelPlanCard = ({ modelID, setStatusMessage }: ModelPlanCardType) => {
+  const { t: modelPlanCardT } = useTranslation('modelPlanCard');
+  const [isExportModalOpen, setIsExportModalOpen] = useState<boolean>(false);
+  const { data, loading } = useGetModelPlanQuery({
+    variables: {
+      id: modelID
+    }
+  });
+
+  const modelPlan = data?.modelPlan || ({} as GetModelPlanTypes);
```

Comment:
This type assertion is dangerous as you destructure twice from this point.  This will cause the app to crash if data is undefined or empty object/{}